include/ui_components/basic/ui_component.hpp

```
@@ -88,14 +78,36 @@ class UIComponent : public UIRenderObject, public UITapHandler, public UITextInp
   void setCursorPosCallback(Offset offset) noexcept override;
 
  protected:
-  virtual bool processChildTaps(const UITapEvent &event) { return false; }
+  virtual bool processChildTaps(const UITapEvent &event) {
+    const auto &children = this->children();
+
+    if (!children.empty() && children.size() > 1) {
```

Clang (14.0.6)

[warning] repeated branch in conditional chain

(bugprone-branch-clone)